browser-extension/mv2/src/client/js/sessionRecorder.js

```diff
@@ -54,7 +55,16 @@ RQ.SessionRecorder.setup = () => {
 };
 
 RQ.SessionRecorder.startRecording = async (options = {}) => {
-  const { config, previousSession, notify, explicit = false, widgetPosition, showWidget, recordingStartTime } = options;
+  const {
+    config,
+    previousSession,
+    notify,
+    explicit = false,
+    widgetPosition,
+    showWidget,
+    recordingStartTime,
+    markRecordingIcon = false,
```

Comment:
```suggestion
    markRecordingIcon = true,
```
We should keep it on by default